libp2p/stream/lpstream.nim

```diff
@@ -47,9 +47,9 @@ type
   LPStreamIncorrectDefect* = object of Defect
   LPStreamLimitError* = object of LPStreamError
   LPStreamReadError* = object of LPStreamError
-    par*: ref CatchableError
+    par*: ref Exception
```

Comment:
you should never have to deal with `Exception` like this - ie a `Defect` should propagate directly at the call site, so only `CatchableError` ever needs storing